runtime/compiler/env/j9method.h

```diff
@@ -242,26 +257,26 @@ class TR_ResolvedJ9MethodBase : public TR_ResolvedMethod
 
    };
 
-
 class TR_ResolvedJ9Method : public TR_J9Method, public TR_ResolvedJ9MethodBase
    {
 public:
    TR_ResolvedJ9Method(TR_OpaqueMethodBlock * aMethod, TR_FrontEnd *, TR_Memory *, TR_ResolvedMethod * owningMethod = 0, uint32_t vTableSlot = 0);
 
-   J9Method *              ramMethod() { return _ramMethod; }
    J9ROMMethod *           romMethod() { return _romMethod; }
-   J9Class *               constantPoolHdr();
-   J9ROMClass *            romClassPtr();
-   J9RAMConstantPoolItem * literals();   // address of 1st CP entry (with type being an entry for array indexing)
+   virtual J9ROMClass *            romClassPtr();
+   virtual J9ConstantPool *      cp();
```

Comment:
Alignment seems to be off for some of these names. Could please align them so that the name of the method starts in the same column?